homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -143,7 +143,13 @@ def __init__(
     def native_value(self):
         """Return the state of the sensor."""
         if self.entity_description.key == "internet":
-            return self.coordinator.data[self.entity_description.key]["kbytes"]
+            try:
+                return self.coordinator.data[self.entity_description.key]["kbytes"]
+            except KeyError:
+                return None
```

Comment:
I think it would be cleaner to use get:
```suggestion
            return self.coordinator.data[self.entity_description.key].get("kbytes")
```
If the `self.entity_description.key` is the key which might be missing you can use:

```suggestion
            return self.coordinator.data.get(self.entity_description.key, {}).get("kbytes")
```

Reply:
It would be clear, to add a `value_fn` lambda to the `SensorEntityDescription` :)

Reply:
I'm trying to help with this one and the level of indirect referencing's getting me spun in a loop, so @frenck ... you're saying this:

```python
SensorEntityDescription(
        key="internet",
        name="Data Used",
        state_class=SensorStateClass.TOTAL_INCREASING,
        native_unit_of_measurement=DATA_KILOBYTES,
        icon="mdi:network",
    )
```

Could be:

```python
SensorEntityDescription(
        key="internet",
        name="Data Used",
        state_class=SensorStateClass.TOTAL_INCREASING,
        native_unit_of_measurement=DATA_KILOBYTES,
        icon="mdi:network",
        value_fn=lambda x,y: x.data[y].get("kbytes"),
    )
```

I tried looking for documentation on how to use but after much googling I just guessed from other people's code...

Reply:
@frenck the lamdas are being implemented, but are not related to this PR/issue

https://github.com/home-assistant/core/pull/65408